src/test/java/com/auth0/client/mgmt/ResourceServerEntityTest.java

```diff
@@ -42,22 +42,18 @@ public void shouldListResourceServers() throws Exception {
 
     @Test
     public void shouldUpdateResourceServer() throws Exception {
+        ResourceServer resourceServer = new ResourceServer("https://api.my-company.com/api/v2/");
+        resourceServer.setId("23445566abab");
```

Comment:
Just keep the new instance line, the pojo class serialization should be tested in a separate file as we did for the rest of them. The scope description serialization for example is never tested. Check the `test/com/auth0/json/mgmt` folder for examples and please add one class for each pojo. You can remove the call to the setters on this "shouldUpdateResourceServer" test.